gender_analysis/corpus.py

```diff
@@ -117,17 +117,18 @@ def _load_documents_and_metadata(self, path_to_files, csv_path):
             all_txt_files = [f for f in os.listdir(path_to_files) if f.endswith('.txt')]
             num_loaded = len(documents)
             num_txt_files = len(all_txt_files)
-            if num_loaded != num_txt_files:
-                # some txt files aren't in the metadata, so issue a warning
-                # we don't need to handle the inverse case, because that
-                # will have broken the document init above
-                print(
-                    f'WARNING: The following .txt files were not loaded because they '
-                    + 'are not your metadata csv:\n'
-                    + str(list(set(all_txt_files) - set(loaded_document_filenames)))
-                    + '\nYou may want to check that your metadata matches your files '
-                    + 'to avoid incorrect results.'
-                )
+            if not ignore_warnings:
+                if num_loaded != num_txt_files:
```

Comment:
This is a _very_ small thing, but it might make things look a little bit nicer if you but these two checks on the same line, as it doesn't appear that there's anything else happening in the outer `if` block